smart-py/marketplace.py

```diff
@@ -35,7 +40,7 @@ def collect(self, params):
             sp.send(self.data.swaps[params.swap_id].creator, sp.utils.nat_to_mutez(self.royalties))
                 
             # send management fees
-            sp.send(self.data.manager, sp.utils.nat_to_mutez(abs(self.fee - self.royalties)))
+            sp.send(self.data.fee_account, sp.utils.nat_to_mutez(abs(self.fee - self.royalties)))
```

Comment:
why change the variable name?

Reply:
To separate it from the manager wallet. It could be useful to have an account for the fees that might be administered by someone else.

In any case. I think the plan from crzy is to set the manager account to a KT contract address that it will have it's own manager. The fees could then be transferred to the KT contract balances (so still sent to self.data.manager). All the "bus accident" code then will be transferred to this new contract.